modules/javafx.web/src/main/java/com/sun/javafx/webkit/prism/WCPathImpl.java

```diff
@@ -344,4 +345,48 @@ public void transform(double mxx, double myx,
         }
         path.transform(BaseTransform.getInstance(mxx, myx, mxy, myy, mxt, myt));
     }
+
+    @Override
+    public boolean strokeContains(double x, double y,
+                                  double thickness, double miterLimit,
```

Comment:
Since corresponding native types are float, the args `thickness, miterLimit, dashOffset, dashArray`  can be declared as float instead of double.